frontend/packages/ceph-storage-plugin/integration-tests-cypress/views/bs.ts

```diff
@@ -0,0 +1,71 @@
+export enum Providers {
+  AWS = 'AWS S3',
+  AZURE = 'Azure Blob',
+  S3 = 'S3 Compatible',
+  PVC = 'PVC',
+}
+
+const ACC_KEY = 'my_dummy_test_key';
+const SEC_KEY = 'my_dummy_sec_key';
+const TARGET = 'my_dummy_target';
+const ENDPOINT = 'http://test-endpoint.com';
```

Comment:
Is these constants are used only in 1 place, then please use them it directly there instead of declaring them as constants.

Reply:
done